app/uk/gov/hmrc/exports/scheduler/Scheduler.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.scheduler
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import javax.inject.{Inject, Singleton}
+import play.api.Logger
+import uk.gov.hmrc.exports.config.AppConfig
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.FiniteDuration
+
+@Singleton
+class Scheduler @Inject()(
+  actorSystem: ActorSystem,
+  appConfig: AppConfig,
+  schedulerDateUtil: SchedulerDateUtil,
+  scheduledJobs: ScheduledJobs
+) {
+
+  scheduledJobs.jobs.foreach { job =>
+    Logger.info(
```

Comment:
I think it'll be good to create logger for Scheduler class and use it, not this from play, because when you create your own logger the log will contain whole information what the log is it, so in this case you'll know that it's Scheduler log.

`private val logger = Logger(classOf[Scheduler])`

Reply:
Good spot.  Thanks

Reply:
I think we need to agree on our approach to this team wide as I've noticed a few different approaches to logging, could you stick a note on the tech debt board please @PatrykRudnicki  :)

Reply:
Yes, we had this metting before, but we can do another one, because team changed :)